File: PYTHON/strukpembelian.py
nama = input("Masukkan nama barang: ")
harga = float(input("Masukkan harga barang: "))
jumlah = int(input("Masukkan jumlah beli: "))

total = harga * jumlah

if total > 10000:
    diskon = total * 0.010
else:
    diskon = 0
    
total_bayar = total - diskon

print("\n==== STRUK PEMBELIAN ====")
print(F"Nama Barang  : {nama}")
print(f"Harga Satuan : {harga}")
print(f"Jumlah Beli  : {jumlah}")
print(f"Total Harga  : {total}")
print(f"Diskon       : {diskon}")
print(f"Total Bayar  : {total_bayar}")
print("============================")

File: JAVA/strukpembelian.java
import java.util.Scanner;

public class strukpembelian {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.print("Masukkan nama barang: ");
        String nama = input.nextLine();

        System.out.print("Masukkan harga barang: ");
        double harga = input.nextDouble();

        System.out.print("Masukkan jumlah beli: ");
        int jumlah = input.nextInt();

        double total = harga * jumlah;
        double diskon = 0;

        if (total > 10000) {
            diskon = total * 0.010;
        }

        double total_bayar = total - diskon;

        System.out.println("\n==== STRUK PEMBELIAN ====");
        System.out.println("Nama Barang  : " + nama);
        System.out.println("Harga Satuan : " + harga);
        System.out.println("Jumlah Beli  : " + jumlah);
        System.out.println("Total Harga  : " + total);
        System.out.println("Diskon       : " + diskon);
        System.out.println("Total Bayar  : " + total_bayar);
        System.out.println("============================");

        input.close();
    }
}
